<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceList</class>
 <widget class="QDialog" name="SequenceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>流程库</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;宋体&quot;;</string>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>流程ID</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>流程名称</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>中文名称</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>创建时间</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>厂站名</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备名</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>添加流程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonModi">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>修改流程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>删除流程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSequenceInto">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>导入流程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSequenceConfig">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>打开流程配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStnDev">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>选择厂站设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefresh">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>刷   新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>退   出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonModi</tabstop>
  <tabstop>pushButtonDel</tabstop>
  <tabstop>pushButtonRefresh</tabstop>
  <tabstop>pushButtonQuit</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
